### DISCUSSION TITLE
Welcome to ungoogled-chromium Discussions!

### DISCUSSION THREAD

**Discussion by Eloston** [Eloston/ungoogled-chromium#1275] [General]
I'm trying out Discussions, a new GitHub feature. The pre-filled content below gives the motivation of Discussions

---

## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  ~~To get started, comment below with an introduction of yourself and tell us about what you do with this community.~~ Maybe we shouldn't do that. Might be a lot of spam.

**Comment by kgwooo** [comment]
Mouse copy/paste problems recently...right button acts as left button when released. This causes repetitive pasting of whatever is in the buffer/clipboard.  Happens in Mint and XFCE.
